package.json

```diff
@@ -22,6 +22,10 @@
   "typescript": {
     "definition": "dist/index.d.ts"
   },
+  "files": [
+    "dist",
+    "COPYING"
```

Comment:
what's `COPYING`?

Reply:
No idea, it was in the package that was built so I assumed it was something in the repo. Happy to remove

Reply:
```suggestion
```

yeah let's remove it then. I don't see this in my local repo and neither in the tarball of 3.x, so I don't think we need it.